pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.gwtbootstrap3</groupId>
         <artifactId>gwtbootstrap3-parent</artifactId>
-        <version>1.5.2</version>
+        <version>1.5.11</version>
```

Comment:
Why the large jump in version?  Could this just be 1.5.3?

Reply:
Correct, 1.5.3.